store/tikv/2pc.go

```diff
@@ -1016,6 +1068,25 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) (err error) {
 	c.stripNoNeedCommitKeys()
 
 	var commitTS uint64
+
+	if c.isOnePC() {
+		if c.onePCCommitTS == 0 {
```

Comment:
Is it safe to read it?

Reply:
Here prewriting must have finished, and no other threads(goroutines) may modify it, I think 🤔
Bug I just notice that the async commit's code [there](https://github.com/pingcap/tidb/blob/ff2e83be642fb61143a8551343e6fa8a8e2e7355/store/tikv/prewrite.go#L189) is suspicious... @sticnarf PTAL...

Reply:
Could you clarify it? Modifying minCommitTS needs acquiring mutex.

Reply:
Oh, that's my mistake. It's locked properly.